src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as quo.theme]
+            [status-im.ui.components.icons.icons :as icons]
```

Comment:
```suggestion
            [quo2.components.icon :as icons]
```
`status-im` namespace contain outdated code

Reply:
yep, avoid using `status-im` and `quo` namespaces where possible :)

Reply:
Done